docs/faq.md

```diff
@@ -1,10 +1,10 @@
 # FAQ
 
-## what is a repository? { id="repositories" }
```

Comment:
Don't remove these section IDs. They are kept to be consistent with the old docs so old links work properly.